### PR TITLE
fix: tailwind-issue

### LINKED ISSUE #1: Tailwind classes are not working
When you run the app, you will get this warning in your console:
```
warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
```
Which means that none of Tailwind utility classes have been found in our files.
When I check the main page, I can notice that the `<h1>` element has no styling, but there are styles defined inside the `index.html` file. 

> main page
image

> index.html
image

Your task is to figure out what the problem is and fix it.
> Hint: Read the warning message and do some research

### PR DESCRIPTION
**Pull request by Bill-1991**
fixes #1

### CODE REVIEW

File: tailwind.config.cjs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+  content: ["./src/**/*.{html,js}",
+            "./**/*.html"],
```
Comment (Dorfieeee) [inline tailwind.config.cjs:4]: This defies the purpose of targeting only the /src directory on the line above.
You are on the right track.
Can you think of a way to target only the one specific file?

**Review by Dorfieeee — CHANGES_REQUESTED** [review]

**Review by AndreyDmitriev39r — CHANGES_REQUESTED** [review]
Please link this PR to the corresponding issue

File: tailwind.config.cjs (hunk 1 above)
Comment (Bill-1991) [reply tailwind.config.cjs:4]: i'll try it...if we take the index.html inside src would be one but can this be done?

Comment (Dorfieeee) [reply tailwind.config.cjs:4]: index.html must stay in the project's root directory

Comment (Bill-1991) [reply tailwind.config.cjs:4]: Yeah...ok

Comment (Dorfieeee) [reply tailwind.config.cjs:4]: the server expects `index.html` in the root directory as an entry point. you can try move it to `/src` and see what happens

Comment (Bill-1991) [reply tailwind.config.cjs:4]: Yeah i know it's not gonna find it,that was a silly question...i'll try it later and see the problem in that thoroughly

Comment (Dorfieeee) [reply tailwind.config.cjs:4]: Do you know what you are saying by `"./**/*.html"` ?

**Review by Dorfieeee — APPROVED** [review]
Well played.

**Review by AndreyDmitriev39r — APPROVED** [review]
LGTM

**Pull request merged**